tools/acrn-manager/README.rst

```diff
@@ -48,6 +48,11 @@ script that will launch a UOS, for example ``launch_UOS.sh``:
    # acrnctl add launch_UOS.sh -U 1
```

Comment:
@taoyuhong @gvancuts @NanlinXie I don't see a "-U" option supported in the acrnctl.c code?  Maybe it was, but is this command example incorrect now?

Reply:
The '-U 1' is the optional args of launch_UOS.sh.

Reply:
Ah. OK.  I was thinking the ``-U 1`` was part of the ``acrnctl add`` command